### PR TITLE
GG-5730 : Update to latest version of sbt-auto-build and drop Artifac…

### PR DESCRIPTION
**Pull request by oarshad**
…tory

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
 
 addSbtPlugin("com.typesafe.play"  % "sbt-plugin"          % "2.7.6")
 
-addSbtPlugin("uk.gov.hmrc"        % "sbt-auto-build"      % "2.13.0")
+addSbtPlugin("uk.gov.hmrc"        % "sbt-auto-build"      % "3.2.0")
         
-addSbtPlugin("uk.gov.hmrc"        % "sbt-git-versioning"  % "2.2.0")
-        
-addSbtPlugin("uk.gov.hmrc"        % "sbt-artifactory"     % "1.13.0")
+addSbtPlugin("uk.gov.hmrc"        % "sbt-artifactory"     % "2.0.0")
```
Comment (martin-armstrong) [inline project/plugins.sbt:12]: SbtArtifactory can go as well

File: build.sbt [hunk 2]
```diff
@@ -2,12 +2,12 @@ import uk.gov.hmrc.DefaultBuildSettings.integrationTestSettings
 import uk.gov.hmrc.SbtArtifactory
 
 val microservice = Project("dynamic-stub", file("."))
-  .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning, SbtArtifactory)
+  .enablePlugins(SbtArtifactory)
```
Comment (martin-armstrong) [inline build.sbt:5]: SbtArtifactory can go as well

**Review by martin-armstrong — CHANGES_REQUESTED** [review]

**Review by martin-armstrong — APPROVED** [review]

**Review by tim-holbrook — COMMENTED** [review]
+1

**Review by tim-holbrook — COMMENTED** [review]
+1

**Pull request merged**